website/client/components/groups/chat.vue

```diff
@@ -1,33 +1,27 @@
 <template lang="pug">
   .row.chat-row
     .col-12
-      h3(v-once) {{ label }}
+      a.float-right(v-markdown='$t("markdownFormattingHelp")')
 
-      .row
-        textarea(:placeholder='placeholder',
-                  v-model='newMessage',
+      h3(v-once) {{ label }}
+      vue-tribute(:options="autocompleteOptions", v-on:tribute-replaced='autocompleteReplaced')
+        .user-entry(:placeholder='placeholder',
+                  @input="updateChatInput"
                   ref='user-entry',
                   :class='{"user-entry": newMessage}',
-                  @keydown='updateCarretPosition',
                   @keyup.ctrl.enter='sendMessageShortcut()',
                   @paste='disableMessageSendShortcut()',
-                  maxlength='3000'
+                  maxlength='3000',
+                  :contenteditable='true'
```

Comment:
this will make the textarea render any type of HTML, I understand why it was added but I feel like it we should add some sort of sanitization or people will be able to copy and paste entiere word docs or html pages and have them rendered

Reply:
Not entirely sure how to handle this (or big of an issue this actually is, since it would only affect how it's displayed in this box and not the actual chat message). I could import a library to handle this sanitizing? Writing that myself might be quite some work.

Reply:
I think the problem is that it would affect the displayed chat as well because markdown renders HTML code as well so it would be rendered. My biggest fear is that it could contain unsanitized JavaScript as well which could result in serious security issues but I haven't tested it